### PR TITLE
Add integration-tests for AsciiDoc

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

**Comment by ia3andy** [comment]
This should fail in native in theory

**Comment by mkouba** [comment]
> This should fail in native in theory

And indeed it failed.

File: asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocConverter.java [hunk 1]
```diff
@@ -13,11 +13,45 @@ public class AsciidocConverter {
...
-        Body body = parser.parseBody(asciidoc, new Parser.ParserContext(null));
+        final String content = trimIndent(asciidoc);
+        Body body = parser.parseBody(content, new Parser.ParserContext(null));
         // Renderer is not thread-safe and must not be shared
         AsciidoctorLikeHtmlRenderer renderer = new AsciidoctorLikeHtmlRenderer(config);
         renderer.visitBody(body);
         return renderer.result();
     }
 
+    public static String trimIndent(String content) {
```
Comment (mkouba) [inline asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocConverter.java:24]: Is this a workaround for https://github.com/yupiik/tools-maven-plugin/issues/21? If so, shouldn't we add a comment here?

Comment (ia3andy) [reply asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocConverter.java:27]: I added a comment, whatever the fix is, we should keep it as explained in the comment.

**Review by mkouba — APPROVED** [review]

**Pull request merged**